packages/miniflare/src/index.ts

```diff
@@ -409,8 +409,22 @@ function getQueueProducers(
 				);
 			}
 
-			for (const [bindingName, opts] of Object.entries(workerProducers)) {
-				queueProducers.set(bindingName, { workerName, ...opts });
+			type Entries<T> = { [K in keyof T]: [K, T[K]] }[keyof T][];
+			type ProducersIterable = Entries<typeof workerProducers>;
+			const producersIterable = Object.entries(
+				workerProducers
+			) as ProducersIterable;
```

Comment:
I think this can just be:

```suggestion
			const producersIterable = Object.entries<typeof workerProducers>(
				workerProducers
			);
```

Reply:
the type of `workersProducers` here is
```
Record<string, string> | Record<string, {
    queueName: string;
    deliveryDelay?: number | undefined;
}>
```

Calling `Object.entries(workersProducers)` will return an array, where each item is an array of type either `[string, string]` or `[string,  {queueName: string; deliveryDelay?: number | undefined;}`.

`Object.entries<typeof workerProducers>(workerProducers)` alone won't give us the typing we need here, as it results in
```
string, Record<string, string> | Record<string, {
    queueName: string;
    deliveryDelay?: number | undefined;
}>][]
```

So we need that extra `Entries` to get the right typing